front-end/yarn.lock

```diff
@@ -0,0 +1,9415 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I don't think this file should be here, we are using npm for the rest of the packages not yarn.